dateparser/date.py

```diff
@@ -268,7 +268,7 @@ class DateDataParser:
 
     :param locales:
         A list of locale codes, e.g. ['fr-PF', 'qu-EC', 'af-NA'].
-        The parser uses locales to translate date string.
+        The parser uses only these locales to translate date string.
```

Comment:
Clarified because of the comment here: https://github.com/scrapinghub/dateparser/pull/789#discussion_r492621092